requirements/requirements_docs.txt

```diff
@@ -1,3 +1,4 @@
+Jinja2<3.1
```

Comment:
When running `bash update_docs_docker.sh` in the docs folder, I was getting this error: https://github.com/readthedocs/readthedocs.org/issues/9038

To fix this, I set the requirement in the docs build.

Reply:
Wow it's still an issue after all this time ?? I could run it last year without this ..

Reply:
Tried again but still required this requirement, as it's building a docker image I'll assume that it would be the same for everyone else